packages/connect-popup/src/types/eth-phishing-detect.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module 'eth-phishing-detect' {
+    export default function isPhishingDomain(domain: string): boolean;
```

Comment:
```suggestion
    export default function (domain: string): boolean;
```
You can import it with whatever name you want, so I would remove the name

Reply:
640a9cf3b6f34e8a421365b2d97912055acf0589